_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -83532,7 +83532,7 @@
 /obj/item/paper{
 	info = "I.O.U. some shinies - Mr V. Ox"
 	},
-/mob/living/simple_animal/hostile/scarybat/batswarm,
+/mob/living/simple_animal/hostile/scarybat,
```

Comment:
........

This is the last time I accept a remap from S34N

Reply:
How was i meant to know it was admin only